grafana-plugin/src/components/ManualAlertGroup/ManualAlertGroup.tsx

```diff
@@ -26,55 +43,137 @@ const ManualAlertGroup: FC<ManualAlertGroupProps> = (props) => {
+            </Alert>
+          ))}
+      </VerticalGroup>
+    );
+  });
+
+  const DirectPagingIntegration = ({ data }) => {
+    return (
+      <VerticalGroup>
+        <Label>Team will be notified according to the integration settings:</Label>
+        <ul className={cx('responders-list')}>
+          <li>
+            <HorizontalGroup justify="space-between">
+              <Text>{data?.verbal_name}</Text>
+              <HorizontalGroup>
+                <PluginLink target="_blank" query={{ page: 'integrations', id: data.id }}>
+                  <IconButton
+                    tooltip="Open integration in new tab"
+                    style={{ color: 'var(--always-gray)' }}
+                    name="external-link-alt"
+                  />
+                </PluginLink>
+              </HorizontalGroup>
+            </HorizontalGroup>
+          </li>
+        </ul>
+      </VerticalGroup>
+    );
+  };
+
   return (
     <>
-      <Drawer scrollableContent title="Create manual alert group" onClose={onHide} closeOnMaskClick={false}>
-        <VerticalGroup spacing="lg">
+      <Drawer
+        scrollableContent
+        title="Create alert group (Direct Paging)"
+        onClose={onHide}
+        closeOnMaskClick={false}
+        width={'70%'}
+      >
+        <VerticalGroup>
+          <GForm form={manualAlertFormConfig} data={data} onSubmit={handleFormSubmit} />
+          <Field label="Select team you want to notify">
+            <GrafanaTeamSelect withoutModal onSelect={onUpdateSelectedTeam} />
+          </Field>
+          <DirectPagingIntegrationVariants
+            selectedTeamId={selectedTeamId}
+            selectedTeamDirectPaging={selectedTeamDirectPaging}
+          />
+
           <EscalationVariants
             value={{ userResponders, scheduleResponders }}
             onUpdateEscalationVariants={onUpdateEscalationVariants}
+            variant={
+              (selectedTeamId && selectedTeamDirectPaging) || userResponders.length || scheduleResponders.length
+                ? 'secondary'
+                : 'primary'
+            }
+            withLabels={true}
           />
-          <GForm form={manualAlertFormConfig} data={data} onSubmit={handleFormSubmit} />
-          {store.teamStore.currentTeam.slack_team_identity && (
-            <Block className={cx('info-block')}>
-              <Icon name="info-circle" />{' '}
-              <Text type="secondary">
-                The alert group will also be posted to #{store.teamStore.currentTeam?.slack_channel?.display_name} Slack
-                channel.
-              </Text>
-            </Block>
-          )}
           <HorizontalGroup justify="flex-end">
             <Button variant="secondary" onClick={onHide}>
               Cancel
             </Button>
             <Button
               type="submit"
               form={manualAlertFormConfig.name}
-              disabled={!userResponders.length && !scheduleResponders.length}
+              disabled={
```

Comment:
I prefer to define local var above if there is some complicated condition like `const submitButtonDisabled = !((selectedTeamId && selectedTeamDirectPaging) || userResponders.length || scheduleResponders.length)`